src/main/java/gov/usgs/owi/nldi/controllers/LinkedDataController.java

```diff
@@ -82,25 +81,34 @@ public LinkedDataController(LookupDao inLookupDao, StreamingDao inStreamingDao,
 		return rtn;
 	}
 
-	@GetMapping(value="linked-data/{featureSource}", produces=MediaType.APPLICATION_JSON_VALUE)
-	public void getFeatures(HttpServletRequest request, HttpServletResponse response,
-							@PathVariable(LookupDao.FEATURE_SOURCE) String featureSource) {
-		BigInteger logId = logService.logRequest(request);
-
-		try {
-			Map<String, Object> parameterMap = new HashMap<>();
+	@GetMapping(value="linked-data/{featureSource}")
```

Comment:
The Spring way would be to leave this method as it was and create a new method with produces=html and the logic specific to that.